.github/workflows/ci.yml

```diff
@@ -180,6 +180,11 @@ jobs:
           build-arch: "amd64"
           build-platform: "linux/amd64"
           tag-suffix: "-ubi-boringssl"
+        - dockerfile: "Dockerfile.standalone"
+          build-args: "CGO_ENABLED=0"
+          build-arch: "amd64 arm64 s390x"
+          build-platform: "linux/amd64,linux/arm64,linux/s390x"
+          tag-suffix: "multiarch" # build for multiarch
```

Comment:
why do we want to use the `Dockerfile.standalone` here and provide a separate tag? 
I think we can just add  `s390x` to the first list element above, there is a multiarch build for it already, i think this should work. WDYT?

Reply:
Oh, I see what should be happening now. Just made the updates :)